src/project-css/app/Http/Controllers/WizardController.php

```diff
@@ -0,0 +1,294 @@
+    *
+    * @return view
+    */    
+    public function import() {
+        return view('admin/wizard/import')->with('AuthUsr', Auth::user());
+    }
+
+    public function importCollection($colID) {
+        // get shared view data
+        $data = $this->sharedViewData;
+        
+        // set collection id to current
+        $data['colID'] = $colID;
+
+        if (Collection::findorFail($colID)->isCms) {
+            // return cms import wizard
+            return view('admin/wizard/cms')->with($data);
+        }
+
+        // return flatfile import wizard
+        return view('admin/wizard/flatfile')->with($data);
+    }
+
+   /**
+    * Returns view for the CMS Import
+    *
+    * @return view
+    */      
+    public function cms() {
+        return view('admin/wizard/cms')->with($this->sharedViewData);
+    }
+
+   /**
+    * Returns view for the FlatFile Import
+    *
+    * @return view
+    */      
+    public function flatfile() {
+        return view('admin/wizard/flatfile')->with($this->sharedViewData);
+    }
+
+   /**
+    * Process form request to create a new table in a collection
+    *
+    * Prepares an array of values required by storeUploadsAndImport
+    * to continue processing the import request.
+    *
+    * @param request The request object has 2 fields that we require for the import
+    * flatFiles should be an array of files and the imprtTblNme (New Table Name)
+    *
+    * @return redirect
+    */       
+    public function flatfileUpload(Request $request) {
+        // 1. Input the table name in the meta Directory
+        //Rules for validation
+        $rules = array(
+            'imprtTblNme' => 'required|unique:tables,tblNme|max:30|min:6|alpha_num',
+            'colID' => 'required|Integer',
+            'fltFile' => 'required|file|mimetypes:text/plain|mimes:txt,dat,csv,tab',
+        );
+
+        //Customize the error messages
+        $messages = array(
+            'imprtTblNme.required' => 'Please enter a table name',
+            'imprtTblNme.unique' => 'The table name has already been taken by current or disabled table',
+            'imprtTblNme.max' => 'The table name cannot exceed 30 characters',
+            'imprtTblNme.min' => 'The table name should be 6 characters or more',
+            'imprtTblNme.alpha_num' => 'The table name can only have alphabets or numbers without spaces',
+            'colID.required' => 'Please select a collection',
+            'colID.Integer' => 'Please select an existing collection',
+            'fltFile.required' => 'Please select a valid flat file',
+            'fltFile.file' => 'Please select a valid flat file',
+            'fltFile.mimetypes' => 'The flat file must be a file of type: text/plain.',
+            'fltFile.mimes' => 'The flat file must have an extension: txt, dat, csv, tab.',
+        );
+
+        // Validate the request before storing the data
+        $this->validate($request, $rules, $messages);
+
+        $data = [
```

Comment:
Again this data is being set similarly to other places in the application, I really feel a helper or abstraction is needed.